app/assets/javascripts/pretty-text/engines/discourse-markdown/code.js

```diff
@@ -37,6 +37,7 @@ export function setup(helper) {
     opts.defaultCodeLang = siteSettings.default_code_lang;
     opts.acceptableCodeClasses = (siteSettings.highlighted_languages || "")
       .split("|")
+      .filter(Boolean)
```

Comment:
This feels too fancy for me... why not the simple `x => x !== ""`

Reply:
This is js idiom, we use it at lots of places in the app already

Reply:
Which is what we want no? It filters anything not truthy, and we don't want anything falsey here so it does exactly what we expect.